docs/docsite/rst/roadmap/COLLECTIONS_5.rst

```diff
@@ -14,23 +14,24 @@ Release schedule
-:2021-10-12: Last day for new collections to be submitted for inclusion in Ansible-5. Collections MUST be reviewed and approved before being included. There is no guarantee that we will review every collection. The earlier your collection is submitted, the more likely it will be that your collection will be reviewed and the necessary feedback can be addressed in time for inclusion.
-:2021-10-13: Community IRC Meeting topic: List any new collection reviews which block release. List any backwards incompatible collection releases that beta1 should try to accommodate.
-:2021-10-19: Ansible-5.0.0 alpha3.
-:2021-10-20: Community IRC Meeting topic: Decide what contingencies to activate for any blockers that do not meet the deadline.
-:2021-10-25: Last day for new collections to be **reviewed and approved** for inclusion in Ansible-5.
-:2021-10-25: Last day for collections to make backwards incompatible releases that will be accepted into Ansible-5.
-:2021-10-26: Create the ansible-build-data directory and files for Ansible-6; new collection approvals will target this.
-:2021-10-26: Ansible-5.0.0 beta1 -- feature freeze [1]_ (weekly beta releases; collection owners and interested users should test for bugs).
-:2021-11-02: Ansible-5.0.0 beta2.
-:2021-11-09: Ansible-5.0.0 rc1 [2]_ [3]_ (weekly release candidates as needed; test and alert us to any blocker bugs).
-:2021-11-16: Ansible-5.0.0 release.
-:2021-12-07: Release of Ansible-5.1.0 (bugfix + compatible features: every three weeks).
+:2021-09-24: ansible-core feature freeze.
+:2021-09-27: start of ansible-core 2.12 betas (weekly, as needed).
+:2021-10-05: Ansible-5.0.0 alpha1 (roughly biweekly ``ansible`` alphas timed to coincide with ``ansible-core-2.12`` pre-releases).
+:2021-10-18: first ansible-core release candidate, stable-2.12 branch created.
+:2021-10-19: Ansible-5.0.0 alpha 2
+:2021-10-26: Last day for new collections to be submitted for inclusion in Ansible-5. Collections MUST be reviewed and approved before being included. There is no guarantee that we will review every collection. The earlier your collection is submitted, the more likely it will be that your collection will be reviewed and the necessary feedback can be addressed in time for inclusion.
+:2021-10-27: Community IRC Meeting topic: List any new collection reviews which block release. List any backwards incompatible collection releases that beta1 should try to accommodate.
+:2021-11-02: Ansible-5.0.0 alpha3
+:2021-11-03: Last day for new collections to be **reviewed and approved** for inclusion in Ansible-5.
+:2021-11-03: Community IRC Meeting topic: Decide what contingencies to activate for any blockers that do not meet the deadline.
+:2021-11-08: Ansible-core-2.12 released.
+:2021-11-08: Last day for collections to make backwards incompatible releases that will be accepted into Ansible-5.
+:2021-11-09: Create the ansible-build-data directory and files for Ansible-6; new collection approvals will target this.
```

Comment:
just asking  - do you really create the Ansible 6 ansible-build-data directory at this point, or is it Ansible-5?

Reply:
Yes.  The idea is that we don't want contributors to be in a position where they have a collection that they've worked on and is approved but it can't be recorded in the build data for several weeks because the versions there are locked for the pending release.  By creating the files for Ansible-6 at this point in time, The new collections can be recorded there even though they have just barely missed the cutoff for Ansible-5.